### PR TITLE
OPNET-8: Added a release note for VMware vSphere - Migrate to using unicast keepalived

### PR DESCRIPTION
**Pull request by ahardin-rh**
Version(s):
4.11 RN only

Addresses:
https://issues.redhat.com/browse/OPNET-8

Link to docs preview:
http://file.rdu.redhat.com/~ahardin/August-2022/4-11-unicast-keepalived/release_notes/ocp-4-11-release-notes.html#ocp-4-11-vsphere-unicast-default

Change management required

### CODE REVIEW

**Comment by cybertron** [comment]
This ended up not being specific to vsphere. Vsphere was the original driver of the change, but since we'd already been running unicast on baremetal for a while we decided to just turn it on for all on-prem platforms to reduce our support matrix, so this applies to vsphere, openstack, and ovirt.

**Comment by ahardin-rh** [comment]
@cybertron Thank you! Updated.

**Comment by ahardin-rh** [comment]
@qiowang721 Can you please provide QE review? Thank you!

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -646,6 +646,11 @@ For clusters that run on {rh-openstack}, the network functions virtualization de
 
 These changes are reflected in simplified post-installation and network configuration documentation.
 
+[id="ocp-4-11-vsphere-unicast-default"]
+==== Installations on Red Hat OpenStack Platform, VMware vSphere, or oVirt now configure keepalived with unicast as the default
```
Comment (jboxman-rh) [inline release_notes/ocp-4-11-release-notes.adoc:650]: @ahardin-rh do we want to use the attribute for OSP here?

**Review by jboxman-rh — COMMENTED** [review]
Left a comment, thanks!

**Comment by jboxman-rh** [comment]
@ahardin-rh, do we need a change management email about this?

**Comment by ahardin-rh** [comment]
@jboxman-rh Yes, as noted above this is going through the change management process. The email is being sent out today. Thanks!

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (ahardin-rh) [reply release_notes/ocp-4-11-release-notes.adoc:650]: Yes, updating now

**Pull request merged**